jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt

```diff
@@ -145,7 +145,7 @@ class ExecuteCommandInShellAction(
     private val container: ContainerDetails
 ) : AnAction(message("ecs.execute_command_run_command_in_shell"), null, null) {
     override fun actionPerformed(e: AnActionEvent) {
-        val sessionManagerInstalled = SessionManagerPluginInstallationVerfication.checkInstallation()
+        val sessionManagerInstalled = SessionManagerPluginInstallationVerification.checkInstallation()
```

Comment:
Minor unrelated nit, since we duplicate the handling of the plugin logic in multiple places and the handling is the same (showing a warning) - I'd probably push them together so you have something like :

```kt
SessionManagerPluginInstallationVerification.requireSessionManager {
  // do thing that requires session manager
}
```

```kt
object SessionManagerPluginInstallationVerification {
  fun requireSessionManager(block: () -> Unit) {
    if( /* session manager plugin verified */ ) {
      block()
    } else {
      SessionManagerPluginWarning(project).show()<br class="Apple-interchange-newline">
    }
  }
}
```

Reply:
Made the change, thanks!